lib/hako/schedulers/ecs_elb_v2.rb

```diff
@@ -141,6 +142,24 @@ def modify_attributes
           end
         end
 
+        new_listeners = @elb_v2_config.fetch('listeners').map { |l| { 'port' => l['port'], 'ssl_policy' => l['ssl_policy'] } }
+        if load_balancer
+          current_listeners = elb_client.describe_listeners(load_balancer_arn: load_balancer.load_balancer_arn).flat_map do |page|
+            page.listeners.map { |l| { 'port' => l[:port], 'ssl_policy' => l[:ssl_policy], 'listener_arn' => l[:listener_arn] } }
```

Comment:
Can't we use accessors? e.g., `'port' => l.port`
Also, I don't think we need to convert Aws::ElasticLoadBalancingV2::Types::Listener  into Hash.